modules/installation/pages/install-uyuni.adoc

```diff
@@ -0,0 +1,78 @@
+[[install-uyuni]]
+= Install Uyuni Server with openSUSE
+
+{productname} Server can be installed on openSUSE.
+
+[NOTE]
```

Comment:
I would take this out of an admonition, and make it part of the stem sentence to the actual table.

Reply:
Ok, I'll submit such a patch separately.